commons-hpcc/pom.xml

```diff
@@ -7,8 +7,8 @@
   <description>Common library for HPCC functionality</description>
 
   <parent>
-    <groupId>org</groupId>
-    <artifactId>hpccsystems</artifactId>
+    <groupId>org.hpccsystems</groupId>
+    <artifactId>HPCC-JAPIs</artifactId>
     <version>7.6.4-SNAPSHOT</version>
```

Comment:
convention--artifact ID should be lower case

Reply:
Thanks, updated.

Reply:
@Michael-Gardner as discussed, we should name this artifact japis rather than hpcc-japis since it's part of the org.hpccsystems namespace.